packages/gestalt/src/Upsell.js

```diff
@@ -70,50 +77,89 @@ const UpsellLink = ({
 
 export default function Upsell({
   dismissButton,
+  imageData,
   message,
   primaryLink,
   secondaryLink,
   title,
 }: Props): Node {
+  const isImage = imageData && imageData.component.type === Image;
```

Comment:
im getting an error here.

can we do 
`const isImage = imageData?.component instanceof Image;`